src/utils/Catalyst/index.ts

```diff
@@ -3,36 +3,32 @@ import isEthereumAddress from 'validator/lib/isEthereumAddress'
 
 import { isSameAddress } from '../../entities/Snapshot/utils'
 
-import { CatalystProfile, DisplayableNameAndAvatar, ProfileResponse } from './types'
+import { CatalystProfile, DclProfile, ProfileResponse } from './types'
 
 const CATALYST_URL = 'https://peer.decentraland.org'
 export const DEFAULT_AVATAR_IMAGE = 'https://decentraland.org/images/male.png'
 
-export function getDisplayableUsername(profile: CatalystProfile | null, user: string) {
+export function getUsernameAndAvatar(profile: CatalystProfile | null, address: string): DclProfile {
```

Comment:
This function is not used anywhere else, export can be removed.
I'd rename this to `getDclProfile` or `generateDclProfile`.